frontend/packages/kubevirt-plugin/src/components/vms/vm-create-yaml.tsx

```diff
@@ -94,10 +92,6 @@ export const VMCreateYAML = (props: any) => {
   return userMode === 'template' ? (
     <CreateVMTemplateYAML {...props} />
   ) : (
-    <VMCreateYAMLLConnected
-      {...(props as any)}
-      kindObj={VirtualMachineModel.kind}
-      plural={VirtualMachineModel.plural}
-    />
+    <VMCreateYAMLConnected {...(props as any)} plural={VirtualMachineModel.plural} />
```

Comment:
Compared to #9258 this is missing prop

```
kindObj={kubevirtReferenceForModel(VirtualMachineModel)}
```

so please double-check, otherwise looks good.